src/main.ts

```diff
@@ -7,8 +7,9 @@ Vue.use(Vuetify);
 
 let v = new Vue({
     el: "#app",
+    vuetify: new Vuetify(),
     template: `
-    <div>
+    <div data-app>
```

Comment:
If you need to add a javascript data attribute at the root level it should probably go here: https://github.com/SOinIDE-SE491/foundation/blob/master/src-ext/extension.ts#L65

Reply:
I tried putting data-app in extension.ts but that didn't work.

Reply:
What's it used for?

Reply:
I used it for the drop down menu. Somehow v-select was not working before.

Reply:
That is really odd. The only thing I found relating to it is this old thread: https://forum.vuejs.org/t/vuetify-data-app-true-and-problems-rendering-v-dialog-in-unit-tests/27495/9

Whatever, if we need it so be it.

Reply:
Yeah, it's strange. Maybe because of the webview.